### PR TITLE
Update: changed the on code

### LINKED ISSUE #120: Change tests to run on pull request - github workflow
Change this line

https://github.com/arshadkazmi42/squash/blob/master/.github/workflows/nodejs.yml#L3

similar to these lines

https://github.com/gh-conf/rhof/blob/master/.github/workflows/nodejs.yml#L3-L5

### PR DESCRIPTION
**Pull request by biohazard2117**
Fixes #120 

Check everything which applies

 I have added the issue number for which this pull request is created.

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -1,6 +1,15 @@
 name: Node CI
 
-on: [push]
+on:
```
Comment (arshadkazmi42) [inline .github/workflows/nodejs.yml:3]: Let's change line 3 - 12 to below 

```
on:
  - push
  - pull_request
```

similar to this https://github.com/gh-conf/rhof/blob/master/.github/workflows/nodejs.yml#L3-L5

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Thank you for taking this up.

Just a small change in the approach. Sorry for giving the wrong approach before.

Can you make this change.

**Comment by biohazard2117** [comment]
So should I delete the changes I did before ? @arshadkazmi42

File: .github/workflows/nodejs.yml (hunk 1 above)
Comment (biohazard2117) [reply .github/workflows/nodejs.yml:3]: The requested changes has been done. **@arshadkazmi42**

**Review by arshadkazmi42 — APPROVED** [review]
LGTM :tada: 

Thank you

**Pull request merged**